tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+    standalone: false
+  ironicConductor:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+    storageClass: local-storage
+    storageRequest: 10G
+  ironicInspector:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic_inspector
+    passwordSelectors:
+      database: IronicInspectorDatabasePassword
+      service: IronicInspectorPassword
+    preserveJobs: true
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    rabbitMqClusterName: rabbitmq
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic-inspector
+    standalone: false
+  passwordSelectors:
+    database: IronicDatabasePassword
+    service: IronicPassword
+  preserveJobs: true
+  rabbitMqClusterName: rabbitmq
+  rpcTransport: json-rpc
+  secret: osp-secret
+  serviceUser: ironic
+  standalone: false
+status:
+  apiEndpoints:
+    ironic:
+      admin: http://ironic-admin-openstack.apps-crc.testing
+      internal: http://ironic-internal-openstack.apps-crc.testing
+      public: http://ironic-public-openstack.apps-crc.testing
+    ironic-inspector:
+      admin: http://ironic-inspector-admin-openstack.apps-crc.testing
+      internal: http://ironic-inspector-internal-openstack.apps-crc.testing
+      public: http://ironic-inspector-public-openstack.apps-crc.testing
+  conditions:
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: Ready
+  # - message: Bootstrap not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: BootstrapReady
+  - message: DB create completed
+    reason: Ready
+    status: "True"
+    type: DBReady
+  - message: DBsync completed
+    reason: Ready
+    status: "True"
+    type: DBSyncReady
+  # - message: Deployment not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: DeploymentReady
+  # - message: Exposing service not started
+  #   reason: Init
+  #   status: Unknown
+  #   type: ExposeServiceReady
+  - message: Input data complete
+    reason: Ready
+    status: "True"
+    type: InputReady
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: IronicAPIReady
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: IronicConductorReady
+  - message: IronicRabbitMqTransportURL disabled
+    reason: Ready
+    status: "True"
+    type: IronicRabbitMqTransportURLReady
+  - message: Service config create completed
+    reason: Ready
+    status: "True"
+    type: ServiceConfigReady
```

Comment:
I think the next logical thing to do would then be to verify connectivity to the service ports.

Reply:
+1, I think we can do by adding this:

```
---
apiVersion: kuttl.dev/v1beta1
kind: TestAssert
namespaced: true
commands:
  - script: |
      curl $(oc get ironics.ironic.openstack.org ironic -o json | jq -r .status.apiEndpoints.ironic.public)
      curl $(oc get ironics.ironic.openstack.org ironic -o json | jq -r .status.apiEndpoints.ironic.admin)
      curl $(oc get ironics.ironic.openstack.org ironic -o json | jq -r .status.apiEndpoints.ironic.internal)
```

Reply:
I have been trying to implement the above but when run in the script in yaml `oc get ironics.ironic.openstack.org ironic` fails with `Error from server (NotFound): ironics.ironic.openstack.org "ironic" not found`.  The command succeeds when running simply on the command line.